pkgs/by-name/ha/hawkthorne-journey/package.nix

```
@@ -0,0 +1,96 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  buildGoModule,
+  love,
+  lua,
+  zip,
+  makeWrapper,
+  makeDesktopItem,
+  tmx2lua,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "hawkthorne-journey";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "hawkthorne";
+    repo = "hawkthorne-journey";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-RhxI2ChkFCBu2FaW2/eHT1KTTjKP++aHDktT+qQ5ooQ=";
+  };
+
+  nativeBuildInputs = [
+    zip
+    makeWrapper
+  ];
+
+  buildInputs = [
+    love
+    lua
+    tmx2lua
+  ];
+
+  buildPhase = ''
+    runHook preBuild
+
+    # Convert TMX maps to Lua
+    for tmxfile in src/maps/*.tmx; do
+      ${tmx2lua}/bin/tmx2lua "$tmxfile"
```

```suggestion
      tmx2lua "$tmxfile"
```
Please put tmx2lua in nativeBuildInputs